hnn_core/tests/test_viz.py

```diff
@@ -21,12 +21,8 @@ def test_network_visualization():
     params.update({'N_pyr_x': 3,
                    'N_pyr_y': 3})
     net = Network(params)
-    plot_cells(net)
```

Comment:
Did you intend to remove `plot_cells()`?

Reply:
oops, indeed. Fixed now